sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - `TokenRequestContext` added the `IsProofOfPossessionEnabled`, `ResourceRequestMethod`, and `ResourceRequestUri` properties to support Proof of Possession tokens ([45134](https://github.com/Azure/azure-sdk-for-net/pull/45134)).
 - `AccessToken` added the `TokenType` property to support distinguishing Bearer tokens from Proof of Possession (PoP) tokens ([45134](https://github.com/Azure/azure-sdk-for-net/pull/45134)).
 - Moved implementation of `Azure.AzureKeyCredential` into `System.ClientModel.ApiKeyCredential` and made `ApiKeyCredential` the base type for `AzureKeyCredential` ([#46128](https://github.com/Azure/azure-sdk-for-net/pull/46128)).
+- `BearerTokenAuthenticationPolicy` now will attempt to handle Continuous Access Evaluation (CAE) challenges, if present, by default ([#46277](https://github.com/Azure/azure-sdk-for-net/pull/46277)).
```

Comment:
At a high level, for my understanding ... what is it that makes this "Core" logic and not "Identity" logic?

Reply:
This is to facilitate "zero touch adoption" for CAE token revocation for all client SDKs that use this policy for Entra ID auth. This means that simply shipping a GA version of each library gets them CAE functionality "for free" without having to implement a custom policy across all clients. The exceptions are clients that already customize this policy today, such as Key Vault and Storage.

Reply:
It's also worth noting that this whole policy type is primarily concerned with identity :)

Reply:
Yeah, no pushback from me, and thanks for the context.  I am seeing a lot of things with new eyes these days, so raising my curiosity about stuff that I hadn't thought much about before.  At some point, I may ask you more questions, but we can do that all another time :)

Reply:
sure thing!